ui/src/common/components/CheckboxAggregation.jsx

```diff
@@ -88,9 +116,9 @@ class CheckboxAggregation extends Component {
 
   renderBucket(bucket) {
     const { selectionMap } = this.state;
-    const { splitDisplayName } = this.props;
-
+    const { splitDisplayName, bucketHelp } = this.props;
     const bucketKey = bucket.get('key');
```

Comment:
const bucketHelpKey = bucketHelp.get(bucketKey); and pass this directly to ``generateTooltip``